deployments/chef/recipes/yum_repo.rb

```diff
@@ -6,6 +6,11 @@
   repo_path = '/etc/yum.repos.d'
 end
 
+rpm_package 'delete-old-yum-key' do
+  package_name 'gpg-pubkey-098acf3b-55a5351a'
```

Comment:
If this doesn't exist, will it gracefully be a noop?

Reply:
Same thing for the other "delete key" operations in the various Management systems.

Reply:
Yes, it will be a noop if the key does not exist.  I also updated the deployment tests to verify that the old key is removed and succeeds if it does not exist.